src/sign.proj

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="dir.props" />
+  <Import Project="..\dir.targets" />
+
+  <Target Name="Build"
+          Condition="'$(SkipSigning)' != 'true' and '$(SignType)' != 'public'">
+
+    <ItemGroup>
+      <SignBuildArgs Include="/p:ArchGroup=$(ArchGroup)" />
+      <SignBuildArgs Include="/p:ConfigurationGroup=$(ConfigurationGroup)" />
+      <SignBuildArgs Include="/p:RuntimeOS=$(RuntimeOS)" />
+      <SignBuildArgs Include="/p:OfficialBuildId=$(OfficialBuildId)" />
+    </ItemGroup>
+    
+    <!-- MicroBuild signing only supports being called from the full framework MSBuild -->
+    <!-- Use the msbuild.exe that is on the %PATH% from run.cmd -->
+    <Exec Command="msbuild.exe sign.builds @(SignBuildArgs, ' ')"
```

Comment:
Well just reading your comment if this is executed via run.cmd then we will be in a VS command prompt.

Reply:
Had the same thought but from reviewing how corefx build.cmd it always invokes the dev cmd prompt in master currently.

Reply:
Yea, we do this in other places as well.  Like in 

https://github.com/dotnet/corefx/blob/d8ae2adab78bc5b4088a18d8cf826867d39774b3/src/Native/build-native.cmd#L154

https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/msbuild.cmd#L1

Do we support doing a full build from outside of `run.cmd`?

Reply:
>Do we support doing a full build from outside of run.cmd?

Not today but as we move to arcade we will be removing run so we will have to figure out a way to ensure we make this call from a VS command prompt. We might end up doing something like https://github.com/dotnet/roslyn-tools/blob/master/src/RepoToolset/tools/Sign.proj#L19.

Reply:
I would hope at that time we could get rid of this completely and use RepoToolset signing functionaltiy?

Reply:
We will use the signing functionality but I suspect it will need to do this stuff until we get a version of the signing tools that can work on .NET Core.